draft-ietf-quic-ack-frequency.md

```diff
@@ -412,9 +412,14 @@ a Reordering Threshold of 2.
 
 ## Expediting Explicit Congestion Notification (ECN) Signals {#congestion}
 
-An endpoint SHOULD send an immediate acknowledgment when a packet marked
-with the ECN Congestion Experienced (CE) {{?RFC3168}} codepoint in the IP
-header is received and the previously received packet was not marked CE.
+If the Ack-Eliciting Threshold is larger than 1, an endpoint SHOULD send
+an immediate acknowledgement when a packet marked with the ECN Congestion
+Experienced (CE) {{?RFC3168}} codepoint in the IP header is received and
+the previously received packet was not marked CE. From there on, if multiple
+CE-marked packets are received in a row or only non-CE-marked packet received,
+the endpoint resumes to sending acknowledgements based on the Ack-Eliciting
+Threshold or max_ack_delay. That means only when a transition from non-CE-marked
+to CE-marked occurs an immediate acknowledgement is sent.
 
```

Comment:
/That means only when a transition from non-CE-marked
to CE-marked occurs an immediate acknowledgement is sent./
This results in sending an immediate acknowledgement only when there is a transition from non-CE-marked
to CE-marked an  is sent./